package.json

```diff
@@ -37,7 +40,9 @@
     "zwitch": "^1.0.0"
   },
   "devDependencies": {
+    "@types/hast": "^2.3.1",
```

Comment:
Were we putting these in deps instead?

Reply:
Yes, if the types depend on a package, this package belongs in the dependencies. In this case, `@types/hast` and `vfile` should be moved into dependencies, since these are imported in `index.d.ts`.

Reply:
updated